snaplet/seed/define-seed.ts

```diff
@@ -3,11 +3,13 @@ import { faker } from '@faker-js/faker';
 import { format } from 'date-fns';
 import { getSeedClient } from './seed-client';
 import { resetDatabase } from './reset-database';
-import { encryptPassword } from '../../src/data/graphql/password';
-import { findUserById } from '../../src/data/user/user.db.datasource';
+import { encryptPassword } from '../../src/core/security/password';
+import { findUserByEmail, updateUserAddress } from '../../src/data/user/user.db.datasource';
+import { getAddresses } from '../../src/data/address/address.db.datasource';
 
-export const seedUsers = async (length?: number) => {
-  const existingUser: User | null = await findUserById(1);
+export const seedDb = async (length?: number) => {
```

Comment:
dessa forma você não precisa modificar o input nas linhas abaixo (geralmente modificar um input é uma má prática e pode resultar em bugs)
```suggestion
export const seedDb = async (length: number = 50) => {
```